### PR TITLE
GEOT-5733 Fixing ECQL.toCQL encoding of Boolean literals

### PR DESCRIPTION
**Pull request by elahrvivaz**

### CODE REVIEW

**Comment by elahrvivaz** [comment]
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=13568

**Comment by aaime** [comment]
Looks good to me

File: modules/library/cql/src/test/java/org/geotools/filter/text/cql2/FilterToCQLTest.java [hunk 1]
```diff
@@ -330,6 +330,11 @@ public void testAttributeIsReservedLowerCase() throws Exception {
         cqlTest("INTERSECTS(\"point\", POINT (1 2))");
     }
 
+    @Test
+    public void testAttributeEuqalsBoolean() throws Exception {
```
Comment (bradh) [inline modules/library/cql/src/test/java/org/geotools/filter/text/cql2/FilterToCQLTest.java]: Nit: `testAttributeEuqalsBoolean()` -> `testAttributeEqualsBoolean()`

Comment (elahrvivaz) [inline modules/library/cql/src/test/java/org/geotools/filter/text/cql2/FilterToCQLTest.java]: thanks, fixed

**Comment by jnh5y** [comment]
Is there anything else needed for this PR to be merged? 

Also, would a backport to the 17.x branch be possible?

**Pull request merged**